### PR TITLE
Add motions documentation

### PR DESCRIPTION
**Pull request by willm30**
This PR adds documentation on the motions feature.

### CODE REVIEW

**Review by jakubcolony — APPROVED** [review]
Would probably save you the explanatory call we had a few months ago if it had been written then 😉 Nice work!

File: archiecture/Motions.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+5. Finalization. Anybody can finalize a motion, and it can only occur once. The result here depends on whether the motion passed or failed. If the motion failed, then as far as the users are concerned, nothing happens when a motion is finalized. However, if the motion passed, then the underlying on-chain action will be executed. Once a motion is finalized, it transitions to the Claim phase.
+
+6. The Claim phase is where anybody who staked the motion can have their stake returned. This applies whether the motion went to a vote or not. In the event a motion went to a vote, those on the winning side will receive a reward above their initial stake, which comes from the penalty applied to the stakes of those on the losing side. This incentivizes people to only stake / object motions that matter to them!
```
Comment (thomassdk) [inline archiecture/Motions.md:56]: How is the penalty calculated? Could you give an example here, similar the others above?

File: archiecture/Motions.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+3. Depending on which phase our motion is in, we may have to perform additional calculations such as calculating staker rewards. For instance, if a motion is staked, but doesn't go to a vote, we need to be able to return the stakers their stake once it finishes. In this case, it will finish when the staking duration elapses, and so again, this can't be calculated in the block-ingestor. 
+
+In summary, this lambda fetches the latest motion state from the chain, and if necessary, performs mutations to the corresponding `ColonyMotion` database entry to ensure it correctly reflects the current state.
+
+Note: "state" here is being used ambiguously. On the one hand, it refers to the phase a motion is in. On the other, it refers to all of the data we need to keep track of in order to display it to the user in the UI.
+
+### Block ingestor
+
+The block ingestor will also mutate the `ColonyMotion` object for a given motion. As indicated above, these mutations are event-driven. Whenever a user interacts with a motion, we can pick up the corresponding event in the block-ingestor and handle it there.
```
Comment (thomassdk) [inline archiecture/Motions.md:90]:
```suggestion
The block ingestor will also mutate the `ColonyMotion` object for a given motion. As indicated above, these mutations are event-driven. Whenever a user interacts with a motion, we can listen to the corresponding event in the block-ingestor and handle it there. 
```

File: archiecture/Motions.md (hunk 1 above)
Comment (willm30) [reply archiecture/Motions.md:56]: Nein, I don't actually know

**Pull request merged**